### ISSUE TITLE
Slack → Glip conversion slightly broken when using newer JWT glip tokens.

### ISSUE THREAD

**Issue by jarthod** [grokify/chathooks#18]
We (the engage digital team) recently updated some of our Glip tokens in our updown.io integration (which is using the Slack to Glip translation provided by chathooks) and noticed the following differences:

Before (old token):
![image](https://user-images.githubusercontent.com/201687/123107246-a61bdb80-d439-11eb-9413-deefd4680e59.png)

After (JWT token):
![image](https://user-images.githubusercontent.com/201687/123107228-a1572780-d439-11eb-891e-10810d449a9f.png)

- The most important missing part here (the website name/url/link) comes from the `author_name` & `author_link` attributes in the Slack webhook payload.
- I also noticed the `pretext` is missing (it's the "ALERT: DOWN" text).
- The `author_image` (which was broken previously in this example because it's a 404) is now missing, it's not a big deal but if it can be fixed it would be nice
- And finally the integration image is no longer showing the updown.io logo (which is in the `icon_url` field of the Slack webhook).

cc @ylecuyer @spuyet @CedricBM @nel

**Comment by grokify** [comment]
I don't think a token is involved but it's certainly possible that the Glip webhook format changed. I'll look into this.

**Comment by grokify** [comment]
Added Updown.io Slack compatible webhook examples.

* Down: [event-example_down.json](https://github.com/grokify/chathooks/blob/master/docs/handlers/updown/event-example_down.json)
* Recovery: [event-example_recovery.json](https://github.com/grokify/chathooks/blob/master/docs/handlers/updown/event-example_recovery.json)

**Comment by nel** [comment]
@jarthod is updown owner & developer, he would know if webhook format had changed.

**Comment by jarthod** [comment]
Indeed they did not change, and the two screenshot I shown are taken at the same time, for the exact same event (so exactly the same payload) but sent to two different glip channels, one using the new token and one using the old.

**Comment by grokify** [comment]
> Indeed they did not change, and the two screenshot I shown are taken at the same time, for the exact same event (so exactly the same payload) but sent to two different glip channels, one using the new token and one using the old.

That's useful info.

Can you paste the webhook URL formats (can obfuscate the webhookId).

**Comment by jarthod** [comment]
Yes, the old token one (first screenshot) looks like this:
```
https://03pe161wla.execute-api.eu-west-3.amazonaws.com/prod/hook?token=EngageDigital-XXXXX&inputType=slack&outputType=glip&url=2aceeb-35e-4c5-908-644d724b0c
```

and the new JWT token one (second screenshot) looks like this:
```
https://03pe161wla.execute-api.eu-west-3.amazonaws.com/prod/hook?token=EngageDigital-XXXXX&inputType=slack&outputType=glip&url=eyJhbGciOiJIUzI1NiInR5I6IkpXVCJ9.eyJvdCI6InUiLCJvI6IjcyMDMzE5ODExIiwiaWQiOiIxMjczNjUjI3In0.G76j93FWcFOPQkB9Bvl5O0GIboWlCglNE9L1iAM
```

(I replaced the webhook token with "EngageDigital-XXXXX" and mangled the glip token slightly by removing some chars)
We only replaced the old glip token by the new one in the same URL

**Comment by grokify** [comment]
Great info @jarthod. Will look into this.

**Comment by grokify** [comment]
Should be resolved in 621285bf1dc42e7038dc677c0c51b353ca96f4f4

Tested locally.

**Comment by grokify** [comment]
v0.7.0 Released.
